.github/workflows/ci.yml

```diff
@@ -95,18 +93,14 @@ jobs:
             Write-Host "::error ::Failed to activate license after multiple attempts."
           }
 
-      - name: Build project
-        run: |
-          unity -quit -batchmode -nographics -logFile - -projectPath samples/unity-of-bugs -buildWindows64Player ../../artifacts/build/game.exe | Out-Default
-
       # The working directory must be set because the project uses
       # relative paths for certain things.
       # Also, we need to provide the path to Unity because
       # the test project references some of its assemblies.
-      - name: Build Sentry package
+      - name: Build Sentry.Unity Solution
         run: |
           dotnet nuget locals all --clear
-          dotnet build -c Release "-p:UnityPath=C:/Program Files/Unity/Editor/Data/Managed"
```

Comment:
msbuild needs to find Unity (we want that logic to be tested here in CI) so not providing the path to it